src/jdb_to_nwb/convert_photometry.py

```diff
@@ -395,9 +601,11 @@ def add_photometry(nwbfile: NWBFile, metadata: dict):
     elif "ppd_file_path" in metadata["photometry"]:
         # Process ppd file from pyPhotometry
         print("Processing ppd file from pyPhotometry...")
-        
+        ppd_file_path = metadata["photometry"]["ppd_file_path"]
+        signals = process_ppd_photometry(ppd_file_path)
```

Comment:
Change returned values and arguments to match the function definition
```suggestion
        sampling_rate, visits = process_ppd_photometry(nwbfile, ppd_file_path)
```